social_django/compat.py

```diff
@@ -0,0 +1,23 @@
+# coding=utf-8
+import six
+import django
+from django.db import models
+
+if django.VERSION >= (2, 0):
+    from django.urls import reverse
+else:
+    from django.core.urlresolvers import reverse
+
+assert reverse
+
+
+def get_rel_model(field):
+    if django.VERSION >= (2, 0):
+        return field.model
+
+    user_model = field.rel.to
+    if isinstance(user_model, six.string_types):
+        app_label, model_name = user_model.split('.')
+        return models.get_model(app_label, model_name)
```

Comment:
Maybe assign `user_model` to this value and let the final return handle the rest?

Reply:
Yeah, that would be better actually. I had a commit where I forgot to return from the other half of the if. Good that we have a test for it now. 😃